src/python/py_continuous_batching_pipeline.cpp

```diff
@@ -138,6 +143,7 @@ void init_continuous_batching_pipeline(py::module_& m) {
             })
         .def_readwrite("m_scores", &GenerationResult::m_scores)
         .def_readwrite("m_status", &GenerationResult::m_status)
+        .def_readwrite("perf_metrics", &GenerationResult::perf_metrics)
```

Comment:
```suggestion
        .def_readwrite("def_readonly", &GenerationResult::perf_metrics)
```
to be aligned with other pipelines